libs/pika/async_mpi/src/mpi_polling.cpp

```diff
@@ -584,6 +587,9 @@ namespace pika::mpi::experimental {
                 polling_status::busy;
         }
 
+        // -------------------------------------------------------------
+        pika::threads::detail::polling_status poll() { return poll_multithreaded(); }
```

Comment:
Is this in preparation for having a `poll_singlethreaded` or similar variant? Or is this is a leftover change that can be reverted?